{
	"about": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-014872.html",
	"title": "getobject における脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-014872.html",
	"description": "getobject には、不特定の脆弱性が存在します。",
	"identifier": "JVNDB-2020-014872",
	"references": [
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28282",
			"source": "CVE",
			"id": "CVE-2020-28282"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2020-28282",
			"source": "NVD",
			"id": "CVE-2020-28282"
		},
		{
			"text": "https://www.whitesourcesoftware.com/vulnerability-database/CVE-2020-28282",
			"source": "関連文書",
			"id": "WhiteSource Vulnerability Database (CVE-2020-28282)"
		},
		{
			"text": "https://www.ipa.go.jp/security/vuln/scap/cwe.html",
			"id": "CWE-Other",
			"title": "その他(CWE-Other)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:getobject_project:getobject",
			"version": "2.2",
			"vendor": "getobject project",
			"product": "getobject"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
		},
		{
			"version": "3.0",
			"score": "9.8",
			"type": "Base",
			"severity": "Critical",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"date": "2021-09-02T18:11+09:00",
	"issued": "2021-09-02T18:11+09:00",
	"modified": "2021-09-02T18:11+09:00"
}
